lms/djangoapps/instructor_task/tasks.py

```diff
@@ -75,11 +75,11 @@ def rescore_problem(entry_id, xmodule_instance_args):
     action_name = ugettext_noop('rescored')
     update_fcn = partial(rescore_problem_module_state, xmodule_instance_args)
 
-    def filter_fcn(modules_to_update):
-        """Filter that matches problems which are marked as being done"""
-        return modules_to_update.filter(state__contains='"done": true')
+    # def filter_fcn(modules_to_update):
+    #     """Filter that matches problems which are marked as being done"""
+    #     return modules_to_update.filter(state__contains='"done": true')
```

Comment:
I believe that it is safe to delete this code. We check for rescorability a bit further down the stack: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor_task/tasks_helper.py#L529